src/Components/Alert/Components/Form/SuggestedFilters.tsx

```diff
@@ -0,0 +1,156 @@
+import ChevronRightIcon from "@artsy/icons/ChevronRightIcon"
+import {
+  Box,
+  Clickable,
+  Flex,
+  Pill,
+  Skeleton,
+  SkeletonBox,
+  SkeletonText,
+  Text,
+} from "@artsy/palette"
+import { handleFieldsWithMultipleValues } from "Components/Alert/Components/Filters/QuickMultipleSelectAlertFilter"
+import { isValueSelected } from "Components/Alert/Helpers/isValueSelected"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import { SearchCriteriaAttributeKeys } from "Components/SavedSearchAlert/types"
+import { SuggestedFiltersFetchQuery } from "__generated__/SuggestedFiltersFetchQuery.graphql"
+import { times } from "lodash"
+import { Suspense } from "react"
+import { graphql, useLazyLoadQuery } from "react-relay"
+interface SuggestedFiltersProps {
+  transitionToFiltersAndTrack: () => void
+}
+export const SuggestedFilters: React.FC<SuggestedFiltersProps> = ({
+  transitionToFiltersAndTrack,
+}) => {
+  const { state, dispatch } = useAlertContext()
+
+  const { artistIDs } = state.criteria
+
+  const data = useLazyLoadQuery<SuggestedFiltersFetchQuery>(
+    suggestedFiltersFetchQuery,
+    {
+      attributes: { artistIDs },
+    }
+  )
+
+  if (!data.previewSavedSearch?.suggestedFilters.length) {
+    return (
+      <Clickable onClick={transitionToFiltersAndTrack} width="100%">
+        <Flex justifyContent="space-between" alignItems={"center"}>
+          <Box>
+            <Text variant="sm-display">Add Filters:</Text>
+
+            <Text variant="sm" color="black60">
+              Including Price Range, Rarity, Medium, Color
+            </Text>
+          </Box>
+
+          <ChevronRightIcon />
+        </Flex>
+      </Clickable>
+    )
+  }
+
+  const suggestedFilters = data.previewSavedSearch?.suggestedFilters.filter(
+    filter => {
+      // Adding this check to make sure we don't add a filter type that's already
+      // selected
+      return !isValueSelected({
+        selectedCriteria: state.criteria[filter.field] || [],
+        value: filter.value,
+      })
+    }
+  )
+
+  return (
+    <Flex justifyContent="space-between" flexDirection={"column"}>
+      <Box>
+        <Text variant="sm-display">Suggested Filters</Text>
+      </Box>
+
+      <Box mt={1}>
```

Comment:
**suggestion**: add a vertical gap here. This is very artificial example because probably there are no such tiny narrow screens, but in case there will be more pills that don't fit in 1 row, this is how it will look like:

<img width="277" alt="Screenshot 2023-12-01 at 13 49 26" src="https://github.com/artsy/force/assets/3934579/2a1b5f54-4a2a-43b5-ab81-056598453a6f">